external/mcp/package.json

```diff
@@ -41,7 +41,7 @@
     "@microsoft/spark.apps": "*",
     "@microsoft/spark.common": "*",
     "@microsoft/spark.dev": "*",
-    "@modelcontextprotocol/sdk": "^1.7.0"
+    "@modelcontextprotocol/sdk": "1.5.0"
```

Comment:
it has always worked for me, lets sync after standup when your back and try to see what the problem could be.

Reply:
Fixed now that https://github.com/modelcontextprotocol/typescript-sdk/pull/254 is merged and released